docs/sphinx/developers/commit-testing.txt

```diff
@@ -4,81 +4,107 @@ Testing code changes
 Automated tests
 ---------------
 
-At the bottom of many commit messages in
-https://github.com/openmicroscopy/bioformats, you will find a few lines
-similar to this:
+The :source:`Bio-Formats testing framework <components/test-suite>` component
+contains most of the infrastructure to run automated tests against the data
+repository.
 
-::
+After checking out source code and building all the JAR files (see
+:doc:`building-bioformats`), switch to the test-suite component and run the tests using the :program:`ant test-automated` target::
 
-    To test, please run:
+    $ cd components/test-suite
+    $ ant -Dtestng.directory=$DATA/metamorph test-automated
 
-    ant -Dtestng.directory=$DATA/metamorph test-automated
+where $DATA is the path to the full data repository.
 
-This shows the command(s) necessary to run automated tests against the
-files likely to be affected by that commit. If you want to run these
-tests, you will need to do the following:
+.. program:: ant test-automated
 
-Clone bioformats.git and checkout the appropriate branch (by following
-the directions on the :devs_doc:`Git usage <using-git.html>`
-page). Run this command to build all of the JAR files:
+Multiple options can be passed to the :program:`ant test-automated` target 
+either by setting the ``testng.${option}`` environment variable or by passing
+:option:`-Dtestng.${option}=value` via the command-line.
 
-::
+.. envvar:: testng.directory
```

Comment:
@sbesson Can't use use a description list as done in https://github.com/openmicroscopy/bioformats/blob/develop/docs/sphinx/developers/cpp/overview.txt#L713 ?
This is the same situation--they aren't actually command-line options--they are internal variables so the option markup is equally invalid.  They can be marked up verbatim in the description list.